PrebidMobile/PrebidMobileRendering/Prebid/PBMCore/ORTB/PBMORTBBid.swift

```diff
@@ -0,0 +1,245 @@
+//
+// Copyright 2018-2025 Prebid.org, Inc.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+    
+
+import Foundation
+
+// 4.2.2: SeatBid
+
+/// A SeatBid object contains one or more `Bid` objects, each of which relates to a specific impression in the bid
+/// request via the `impid` attribute and constitutes an offer to buy that impression for a given `price`.
+class PBMORTBBid<ExtType: PBMJsonCodable>: PBMJsonCodable, PBMORTBExtensible {
+    /// [Required]
+    /// Bidder generated bid ID to assist with logging/tracking.
+    var bidID: String
+    
+    /// [Required]
+    /// ID of the Imp object in the related bid request.
+    var impid: String
```

Comment:
The general question - what will happen if live OpenRTB don't have this or another field which is non optional? 

I'd not really on OpenRTB notation in implementation and expect that any field can be absent in the response or request. It will make solution more general and robust.